partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/AstPass.kt

```diff
@@ -12,7 +12,7 @@
  *  language governing permissions and limitations under the License.
  */
 
-package org.partiql.planner.internal.normalize
```

Comment:
(self-review) unified `normalize` and `transforms` sub-packages to be under `transforms`. confusingly, we had some normalize code split between the two packages